.github/workflows/publish_download_tracker.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish Download Tracker
+on:
+  release:
+    types: [published]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  publish-download-tracker:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+        # with:
+        #   ref: 'stable'
```

Comment:
I will uncomment this before merging because we want it to install the rdt release version